packages/core-ethereum/src/account.spec.ts

```diff
@@ -1,20 +1,24 @@
 import type { HoprToken } from './tsc/web3/HoprToken'
 import type { Await } from './tsc/utils'
 import type CoreConnector from '.'
-import assert from 'assert'
+import { expect } from 'chai'
```

Comment:
Is there a reason to use `chai` instead of `assert`?

Reply:
easier to read and build tests with